packages/block-editor/src/components/index.js

```diff
@@ -126,6 +126,7 @@ export {
 export { default as DefaultBlockAppender } from './default-block-appender';
 export { default as __unstableEditorStyles } from './editor-styles';
 export { default as Inserter } from './inserter';
+export { searchItems } from './inserter/search-items';
```

Comment:
I think "searchItems" here is confusing and I actually wonder if it should be used in this PR. Basically this function is used to "search inserter items" which are not exactly block types:

 - A reusable block type with initial attributes is a search item
 - A block variation is a search item
 - A block type is a search item
 - I wonder if a pattern is a search item or if it has a different search function.

In other words, it seems too specific to the inserter for me, maybe we could have a "searchBlockType" function or something like that, I also don't see why it should be in "block-editor" instead of being a "blocks" thing. (The inserter search makes sense in block-editor though)

Reply:
It can also be a selector in `core/blocks` package (we do have selectors to fetch block types there)

Reply:
>I think "searchItems" here is confusing and I actually wonder if it should be used in this PR.

Agreed! A selector might be better here and I think it already exists... I'll update soon.

>I wonder if a pattern is a search item or if it has a different search function.

For the record it's used for patterns in the inserter too.

Reply:
Updated to use `isMatchingSearchTerm` selector. Thanks Riad for the feedback!